src/script/campaign/mod/vco-listeners-tmb.lua

```diff
@@ -21,7 +21,7 @@ local function add_listeners()
 				context:mission():mission_record_key():sub(1,26) == "wh2_dlc09_books_of_nagash_";
 		end,
 		function(context)
-			vlc.nagash_books:add(FACTION_ARK_ID, 9);
+			vlc.nagash_books:add(FACTION_ARK_ID, 8);
```

Comment:
I feel like this is the only line that actually needed a change. Everything else is redundant. But it is done already, so approved.

Reply:
Nope - you still can't pass faction id to `complete_mission` - it requires a faction key. No need to have id and key floating around for the same thing - replace all with key.